### DISCUSSION TITLE
Iphone issue with chrome

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#3094] [General Questions & Discussion] [answered]
Have you notice that we can' t open hubs in iphone till yesterday ? 
I have try with 3 iphone and differents versions of iphone, and with chrome , we have an error message  : 
Media devices unsupported . 

Thx for you reply

**Comment by davegoopot** [comment]
@bemyevent when you say "till yesterday", do you mean that Chrome is now working on the iPhone, or is it still broken?

**Reply by bemyevent** [reply]
No it is not working with chrome . I have Some texture loading issu in safari with ios . Have you already see this issu ?

**Comment by davegoopot** [comment]
Yes I see the same. Neither Chrome nor Firefox works for me on iOS.

**Comment by emclaren** [comment, marked as answer]
Hi there, thanks for reaching out about this.

Chrome on iOS is not actually Chrome. It is Safari (Webkit) wrapped in Chrome UI. Apple does not allow browser vendors to run their own browser engines on iOS. They force you to use Webkit, and more specifically, a version of Webkit that does not have the full feature set, like access to the microphone. Since Chrome & Firefox on iphones don't have access to the device microphone, it's not possible to run Hubs on these browsers. The error message should direct users to try to open Hubs using Safari however. If you are not seeing a message directing you to Safari, please let us know so I can file a bug report.
